dashboard/config/scripts/aaron_dummy_md_level.external

```diff
@@ -1,6 +1,7 @@
 name 'Aaron Dummy md level'
 title 'Aaron Dummy md level'
 description 'description here'
+associated_blocks 'studio'
```

Comment:
This level does not look like it is used anywhere. Why are you updating it here?

Reply:
It was an example level displaying this issue. I can remove the change from this PR.